src/client/gui/lib/platform/linux.dart

```diff
@@ -32,6 +33,12 @@ class LinuxPlatform extends MpPlatform {
             CopySelectionTextIntent.copy,
         SingleActivator(LogicalKeyboardKey.keyV, control: true, shift: true):
             PasteTextIntent(SelectionChangedCause.keyboard),
+        SingleActivator(LogicalKeyboardKey.equal, control: true):
```

Comment:
I was going to say this should be `LogicalKeyboardKey.plus` (which doesn't exist) since this breaks for certain keyboard layouts, but it seems that Flutter doesn't provide an easy way to handle this. Do you think this is worth fixing?

Example broken keyboard layout:
1. Set your keyboard layout to Japanese
2. Notice `+` and `=` are on separate keys
3. Notice `ctrl`+`+` and `ctrl`+`=`, don't work since shift is pressed

Reply:
Good catch! Yes, I do think it is worth fixing. I checked other terminals and they handle it correctly, so we should as well.

Reply:
Alright, so from my testing it seems that other apps handle both `+` and `=` as zoom increase, even if they are on separate keys and both require shift. And also they seem to handle numpad plus and minus. So I added those.

Reply:
Well spotted, this would never have occurred to me. Good job guys!